### ISSUE TITLE
[Edge] Can't open text alternative balloon in a table

### ISSUE THREAD

**Issue by Mgsy** [ckeditor/ckeditor5#1995]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

Latest master

## 📋 Steps to reproduce

1. Upload/paste an image to a table cell.
2. Focus the image.
3. Open the text alternative balloon.

## ✅ Expected result
The balloon shows up.

## ❎ Actual result
The editor crashes.

## 📃 Other details that might be useful

**GIF**
![bug_cke5](https://user-images.githubusercontent.com/20988892/63680608-572c9d00-c7f4-11e9-81e2-b020f1a02e7b.gif)

**Error**
```
contextualballoon-remove-view-not-exist: Cannot remove the configuration of a non-existent view. Read more: https://ckeditor.com/docs/ckeditor5/latest/framework/guides/support/error-codes.html#error-contextualballoon-remove-view-not-exist
contextualballoon.js (227,1)
```

**Comment by FilipTokarski** [comment]
This bug still exists on old Edge, with errors: 
```
0: Unable to get property 'isPersonal' of undefined or null reference
0: Unable to get property 'SavePersonalAndPaymentData' of undefined or null reference
```

However it is fixed in the new upcoming version (79.0.309.56)

**Comment by Mgsy** [comment]
Fixed in the new Edge release.
